src/System.Linq/tests/AggregateTests.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+using System;
+using System.Linq;
+
+namespace Test
+{
+    public class AggregateTests
+    {
+        //
+        // Aggregate
+        //
+        [Fact]
+        public static void RunAggregationTests()
+        {
+            RunAggregationTest1_Sum1(1024);
+            RunAggregationTest1_Sum2(1024);
+            RunAggregationTest1_Sum3(1024);
+            RunAggregationTest2(16);
+        }
```

Comment:
>RunAggregationTest1_Sum1(1024);
RunAggregationTest1_Sum2(1024);
RunAggregationTest1_Sum3(1024);
RunAggregationTest2(16);

I think it would be better mark these methods as `Fact` and remove `RunAggregationTests`